### DISCUSSION TITLE
Im new and idk what connect client means

### DISCUSSION THREAD

**Discussion by Hypertex1000** [bedrock-tool/bedrocktool#79] [Q&A]
what does it mean when it says connect client?

**Comment by olebeck** [comment]
you have to open the game and add a new server with the address 127.0.0.1 or localhost
then join it
you might also need to do this
https://doc.pmmp.io/en/rtfd/faq/connecting/win10localhostcantconnect.html

**Reply by Hypertex1000** [reply]
so im trying to download a realm, so i need to join it, when it needs me to input server address i use one of those 2? also what does the authentication do?
